### PR TITLE
Retrieve branch test-driver.yaml for hypershift kubevirt-csi

### PR DESCRIPTION
**Pull request by awels**
Use the OCP server version to locate the correct
release branch and download the appropriate
release version of the test-driver.yaml file.

### CODE REVIEW

File: ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml [hunk 1]
```diff
@@ -14,8 +14,14 @@ chain:
...
-        echo "Getting test-driver.yaml from the master branch"
-        curl -L -o ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST} https://raw.githubusercontent.com/openshift/kubevirt-csi-driver/master/hack/test-driver.yaml
+        version=$(oc version | grep "Server Version: " | awk '{ print $3 }' | sed 's/\.[^.]*$//')
+        if [ -z $version ]; then
+          version="master"
+        else
+          version="release-${version}"
+        fi
+        echo "Getting test-driver.yaml from the ${version} branch"
+        curl -L -o ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST} https://raw.githubusercontent.com/openshift/kubevirt-csi-driver/${version}/hack/test-driver.yaml
```
Comment (davidvossel) [inline ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml:24]: what about the rwx test-driver?

Comment (awels) [reply ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml:24]: That one actually already had code to look up the version. I am aligning this code with that one. Give me a second to update.

File: ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml [hunk 2]
```diff
@@ -12,10 +12,16 @@ chain:
...
       PR_SHA=$(echo ${JOB_SPEC} | jq -r '.refs.pulls[0].sha')
       echo "Source git commit is ${PR_SHA}"
+      ocp_version=$(oc get clusterversion version -o jsonpath='{.status.desired.version}' | awk -F "." '{print $1"."$2}')
       response=$(curl -L --head --write-out '%{http_code}' --silent --output /dev/null https://raw.githubusercontent.com/openshift/kubevirt-csi-driver/${PR_SHA}/hack/test-driver.yaml)
       if [ $response != "200" ]; then
-        echo "Getting test-driver.yaml from the master branch"
-        curl -L -o ${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST} https://raw.githubusercontent.com/openshift/kubevirt-csi-driver/master/hack/test-driver.yaml
+        if [ -z $ocp_version ]; then
+          ocp_version="master"
+        else
```
Comment (davidvossel) [inline ci-operator/step-registry/hypershift/kubevirt/run-csi-e2e/hypershift-kubevirt-run-csi-e2e-chain.yaml:20]: the rwx test driver doesn't have this line. is it important

**Review by davidvossel — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**